crates/swc/Cargo.toml

```diff
@@ -25,7 +25,7 @@ concurrent = [
 ]
 
 debug = ["swc_ecma_visit/debug", "swc_ecma_minifier/debug"]
-default = ["es3", "lint"]
+default = ["es3", "lint", "swc_typescript"]
```

Comment:
Can you rename the feature to `isolated_dts`? (or `isolated-dts` if it's more consistent)